docs/src/paradox/03-apis/api-v2/editing-resources.md

```diff
@@ -173,6 +173,67 @@ The response is a JSON-LD document containing a
 @ref:[preview](reading-and-searching-resources.md#get-the-preview-of-a-resource-by-its-iri)
 of the resource.
 
-## Modifying a Resource
+## Modifying a Resource's Values
 
-To modify a resource, you can modify its values; see @ref:[Editing Values](editing-values.md).
+See @ref:[Editing Values](editing-values.md).
+
+## Modifying a Resource's Metadata
+
+You can modify the following metadata attached to a resource:
+
+- label
+- permissions
+- last modification date
+
+To do this, use this route:
+
+```
+HTTP PUT to http://host/v2/resources
```

Comment:
Shouldn't this be `http://host/v2/resources/[ID]`? I thought our convention was to have the ID in the URL for something that we wanted to change?

Reply:
With JSON-LD, the IRI of the thing you want to change is in the body of the request, so it would be redundant to also put it in the URL. For example, this is how you update a value:

https://docs.knora.org/paradox/03-apis/api-v2/editing-values.html#updating-a-value